hawc/apps/vocab/api.py

```diff
@@ -92,6 +93,18 @@ def related_entity(self, request: Request, pk: int | None = None) -> Response:
         return Response(serializer.data, status=status.HTTP_201_CREATED)
 
 
+class EhvTermViewSet(VocabTermViewSet):
```

Comment:
I had the option of invoking the parent, VocabTermViewSet, with arguments, but I created each subclass to make calling the vocabulary cleaner, which seems to be the method used in other parts of hawc.